regexes/bots.yml

```diff
@@ -61,6 +61,13 @@
     name: 'Alexa Internet'
     url: 'https://www.alexa.com'
 
+- regex: 'Amazonbot'
+  name: 'Amazon web crawler'
```

Comment:
* Set name `Amazon Bot`
* You need to add test to file device-detector/Tests/fixtures/bots.yml

You can get test fixture via the console
To do this, run the command
```shell
cd device-detector/misc
php test.php "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/600.2.5 (KHTML, like Gecko) Version/8.0.2 Safari/600.2.5 (Amazonbot/0.1; +https://developer.amazon.com/support/amazonbot)"
```

Reply:
```
PHP Warning:  require(/mnt/dietpi_userdata/device-detector/misc/../vendor/autoload.php): Failed to open stream: No such file or directory in /mnt/dietpi_userdata/device-detector/misc/test.php on line 13
PHP Fatal error:  Uncaught Error: Failed opening required '/mnt/dietpi_userdata/device-detector/misc/../vendor/autoload.php' (include_path='.:/usr/share/php') in /mnt/dietpi_userdata/device-detector/misc/test.php:13
Stack trace:
#0 {main}
  thrown in /mnt/dietpi_userdata/device-detector/misc/test.php on line 13
```
I guess this needs to be done within the Matomo directory? Found `/matomo/vendor/matomo/device-detector`, but it does not contain the `misc/test.php`. Hehe sorry I'm a bit inexperienced with PHP development. I'll try to apply the test manually.

Reply:
Done. Btw, is there a specific reason why in `regexes/bots.yml`, all strings are single quoted while in `Tests/fixtures/bots.yml`, those are not?

Reply:
> Btw, is there a specific reason why in regexes/bots.yml, all strings are single quoted while in Tests/fixtures/bots.yml, those are not?

There are many special characters for a'string in regular expressions, so they are in quotes,
also all regex are written by hand.
Most all fixtures are created via `misc/test.php` or sorted `misc/sortTestFixtureFiles.php` where extra quotes are removed.

Reply:
@MichaIng Thx you for PR.

Reply:
Thanks for the quick review and merged 👍.